src/zjs_performance.c

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016, Linaro Limited.
+#ifdef BUILD_MODULE_PERFORMANCE
+
+// ZJS includes
+#include "zjs_util.h"
+
+static jerry_value_t zjs_performance_now(const jerry_value_t function_obj,
+                                         const jerry_value_t this,
+                                         const jerry_value_t argv[],
+                                         const jerry_length_t argc)
+{
+    if (argc != 0)
+        return zjs_error("performance.now: no args expected");
```

Comment:
I'd think that error messages are targeted at users, not developers. If you confirm that it's ok to leak internal things like C function names to users and potentially confuse them, I'll be happy to comply, though given the previous comment, this code may be removed at all, so need re-confirming anyway.

Thanks for the review!